shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagerClientImpl.java

```diff
@@ -53,6 +55,15 @@ public Uni<List<BridgeDTO>> fetchBridgesToDeployOrDelete() {
                 .onItem().transform(this::getBridges);
     }
 
+    @Override
+    public Uni<HttpResponse<Buffer>> notifyBridgeStatus(List<BridgeStatusDTO> bridgeStatusDTOs) {
+        LOGGER.debug("Notifying manager about the new status of the Bridge");
+        return getAuthenticatedRequest(webClientManager.put(V2APIConstants.V2_SHARD_API_BASE_PATH), request -> request.sendJson(bridgeStatusDTOs))
+                .onItem().invoke(success -> updateManagerRequestMetricsOnSuccess(MetricsOperation.OPERATOR_MANAGER_UPDATE, success))
```

Comment:
This changes the semantics of metrics handling.

We used to store a metric count of for each resource updated (because we sync'ed each individually). If we are sending status updates for multiple resources in a batch we need to either accept that metrics now represent the number of HTTP updates and not the number of resource updates; or change how metrics are recorded in this new code.

Reply:
How this metric is different from metric recored in `fetchBridgesToDeployOrDelete`. In `fetchBridgesToDeployOrDelete` as well we are recording metric as a single event for a group of BridgeRequest. I am just trying to keep thing consistent.

Reply:
But inconsistent with how `v1` worked. We should at least agree `v1` was wrong and what you've done is correct.

Reply:
there is nothing wrong or right here. In `v1` we are independently sending status back to manager for each bridge whereas in `v2` we are sending them in batch.

I could do following :
 - remove `metric` logic completely from here and create a new Jira to discuss which `metric` are required here.
 - I could move record `mertic` functionality before making REST call. In that case I could iterate over `List<BridgeStatusDTO>` and generate metric foe each for them.
 - Or just keep it as it is.
 
 Please suggest what I should do here.

Reply:
@vaibhavjainwiz I think we should just leave it as it is 👍 

This metric has always simply recorded the number of HTTP requests between data plane and control plane, recording the http status code in the event of failure. For `v1` it would record multiple metrics for each Bridge/Processor because we had the various lifecycle states e.g `PROVISIONING`, `READY`...

In `v2` it will simply record a metric each time we do the sync back. It's not being used to record the number of correctly provisioned resources.